builtin/credential/okta/backend_test.go

```diff
@@ -88,6 +97,57 @@ func TestBackend_Config(t *testing.T) {
 	})
 }
 
+func createOktaGroups(t *testing.T, username string, token string, org string) []string {
+	orgURL := "https://" + org + "." + previewBaseURL
+	ctx, client, err := okta.NewClient(context.Background(), okta.WithOrgUrl(orgURL), okta.WithToken(token))
+	require.Nil(t, err)
+
+	users, _, err := client.User.ListUsers(ctx, &query.Params{
+		Q: username,
+	})
+	require.Nil(t, err)
+	require.Len(t, users, 1)
+	userID := users[0].Id
+	var groupIDs []string
+
+	for i := 0; i < 201; i++ {
```

Comment:
A comment linking to the docs or sdk that define the limit that this is derived from would be good.

Reply:
Done